crates/ide-assists/src/utils.rs

```diff
@@ -84,6 +84,12 @@ pub fn test_related_attribute(fn_def: &ast::Fn) -> Option<ast::Attr> {
     })
 }
 
+#[derive(Clone, Copy, PartialEq)]
+pub enum IgnoreAssocItems {
+    HiddenDocAttrPresent,
```

Comment:
This reads a little backwards to me, but it doesn't matter too much.

Reply:
I am bad at naming stuff. Happy to change it if you have sth in mind.

Reply:
`DocHiddenAttrPresent` :smile:

Reply:
:D ok will do in a bit.